mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/StoredRecordChunksKafkaHandler.java

```diff
@@ -154,19 +154,22 @@ private JsonArray buildJournalRecords(List<Record> storedRecords, Optional<JsonO
     }
 
     for (Record record : storedRecords) {
-      JournalRecord journalRecord = new JournalRecord()
-        .withJobExecutionId(record.getSnapshotId())
-        .withSourceRecordOrder(record.getOrder())
-        .withSourceId(record.getId())
-        .withEntityType(entityType)
-        .withEntityId(record.getId())
-        .withActionType(CREATE)
-        .withActionStatus(COMPLETED)
-        .withActionDate(new Date())
-        .withTitle(allNotNull(record.getParsedRecord(), titleFieldTag)
-          ? ParsedRecordUtil.retrieveDataByField(record.getParsedRecord(), titleFieldTag, subfieldCodes) : null);
-
-      journalRecords.add(JsonObject.mapFrom(journalRecord));
+
+      if (record.getErrorRecord() == null) {
```

Comment:
@okolawole-ebsco No. All needed logic will be triggered in the next data-import-flow steps. This condition is just required for NOT building journal record if there are some errors (because there can be NullPointerExceptions when trying to work with data from mrc-record)

Reply:
understood.